dlpy/model.py

```diff
@@ -770,7 +770,8 @@ def plot_training_history(self, items=['Loss', 'FitError'], fig_size=(12, 5), ti
             raise DLPyError('model.fit should be run before calling plot_training_history')
 
     def evaluate(self, data, text_parms=None, layer_out=None, layers=None, gpu=None, buffer_size=None,
-                 mini_batch_buf_size=None, top_probs=None, use_best_weights=False):
+                 mini_batch_buf_size=None, top_probs=None, use_best_weights=False,
+                 random_crop='none', random_flip='none',  random_mutation='none'):
```

Comment:
are we passing 'none' or None?

Reply:
In this case, none, since we want to overwrite the options from the model DAG.